### PR TITLE
Added Suraj as a current mainatainer

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # .

#### Changes proposed in this Pull Request:
- Add a new contributor to Contributors list on home page
- Add a new team member to Our Team page

#### Screenshots:
**Our Team Safari Screenshot**
Screenshot 2021-09-27 at 8 51 22 PM

**Our Team Chrome Screenshot**
Screenshot 2021-09-27 at 8 52 09 PM

**Contributors Safari Screenshot**
safari-contributors-ss

** Contributors Chrome Screenshot**
chrome-contributors-ss

#### To Test:
**Locally the changes can be seen on the following urls:**
- http://localhost:3000
- http://localhost:3000/community/our-team

### CODE REVIEW

**Comment by daileytj** [comment]
The work itself looks great! Good job Suraj.

One comment... although we are open-source, we still represent Eaton. I'm not sure if it would be professional to advertise being "a party animal" or mentioning that you "party harder". Just my 2 cents... @jeffvg @joebochill @emclaug2 @huayunh feel free to correct me if this is ok in your books.

File: src/__configuration__/contributors/currentMaintainers.ts [hunk 1]
```diff
@@ -57,4 +58,14 @@ export const currentMaintainers: CurrentMaintainter[] = [
             linkedIn: 'https://www.linkedin.com/in/ekta-ghag-138120134/',
         },
     },
+    {
+        name: 'Suraj Karambe',
+        role: `Front-End Developer`,
+        image: SurajKarambe,
+        description: `Suraj is a techie and a party animal, who loves to work hard and party harder. He loves to trek and travel and has his roots in India. He is a big fan of TV serial FRIENDS and a big foodie, who thinks about food all the time.`,
```
Comment (daileytj) [inline src/__configuration__/contributors/currentMaintainers.ts:65]:
```suggestion
        description: `Suraj is a techie and a party animal, who loves to work hard and party harder. He loves to trek and travel and has his roots in India. He is a big fan of TV series FRIENDS and a big foodie, who thinks about food all the time.`,
```

**Comment by jeffvg** [comment]
> The work itself looks great! Good job Suraj.
> 
> One comment... although we are open-source, we still represent Eaton. I'm not sure if it would be professional to advertise being "a party animal" or mentioning that you "party harder". Just my 2 cents... @jeffvg @joebochill @emclaug2 @huayunh feel free to correct me if this is ok in your books.

Yes agree...

**Comment by bkarambe** [comment]
> The work itself looks great! Good job Suraj.
> 
> One comment... although we are open-source, we still represent Eaton. I'm not sure if it would be professional to advertise being "a party animal" or mentioning that you "party harder". Just my 2 cents... @jeffvg @joebochill @emclaug2 @huayunh feel free to correct me if this is ok in your books.

Thanks Tom. I have updated the PR.

**Review by huayunh — APPROVED** [review]

**Review by jeffvg — APPROVED** [review]

**Pull request merged**